### PR TITLE
Remove test for IE

### PR DESCRIPTION
**Pull request by bualoy-napat**
## Description

All test case for IE should be remove

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Refactor (improves code without changing its functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: packages/elements/src/datetime-picker/__test__/datetime-picker.snapshot.test.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ describe('datetime-picker/DOMStructure', () => {
     it('DOM structure is correct when opened', async () => {
       const el = await fixture('<ef-datetime-picker lang="en-gb" view="2020-04" opened></ef-datetime-picker>');
       await nextFrame();
-      await nextFrame(); /* second frame required for IE11 as popup opened might not fit into one frame */
+      await nextFrame();
```
Comment (Sakchai-Refinitiv) [inline packages/elements/src/datetime-picker/__test__/datetime-picker.snapshot.test.js:18]: this frame should be removed

File: packages/elements/src/overlay/__test__/managers/focus-manager.test.js [hunk 2]
```diff
@@ -73,7 +73,7 @@ describe('overlay/manager/FocusManager', () => {
           expect(size()).to.equal(0, 'element should be deregistered');
         });
 
-        xit('Test with activeElement', async () => {
+        it('Test with activeElement', async () => {
```
Comment (Sakchai-Refinitiv) [inline packages/elements/src/overlay/__test__/managers/focus-manager.test.js:76]: xit is skip all browsers, is it works for modern browsers?

File: packages/elements/src/select/__test__/select.navigation.test.js [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
+import { fixture, expect, elementUpdated, aTimeout, nextFrame } from '@refinitiv-ui/test-helpers';
 import { getOptions, openedUpdated, getData, getMenuEl } from './utils';
 
 const keyBoardEvent = async (el, key, options = {}) => {
-  getMenuEl(el).dispatchEvent(fireKeyBoardEvent('keydown', Object.assign({ key }, options)));
+  getMenuEl(el).dispatchEvent(new KeyboardEvent('keydown', Object.assign({ key }, options)));
   await elementUpdated(el);
   await nextFrame();
-  await nextFrame(); // need this for IE11 to ensure focus is set
+  await nextFrame();
```
Comment (Sakchai-Refinitiv) [inline packages/elements/src/select/__test__/select.navigation.test.js:8]: Should remove this frame.

Comment (bualoy-napat) [reply packages/elements/src/select/__test__/select.navigation.test.js:8]: It was failed when I remove this frame

File: packages/translate/__test__/elf-translate-document.test.js [hunk 4]
```diff
@@ -19,7 +19,7 @@ describe('Elf Translate Document Lang Test', () => {
     document.documentElement.lang = 'en';
     await elementUpdated(el);
     await elementUpdated(enUSEl);
-    await nextFrame(); // need for IE11
+    await nextFrame();
```
Comment (Sakchai-Refinitiv) [inline packages/translate/__test__/elf-translate-document.test.js:22]: Should remove this line

File: packages/translate/__test__/elf-translate-element.test.js [hunk 5]
```diff
@@ -47,7 +47,7 @@ describe('Elf Translate Element Lang Test', () => {
     expect(el.defaultEl.innerText).to.equal('Региональные настройки: ru');
     el.lang = 'en';
     await elementUpdated(el);
-    await nextFrame(el); // need for IE11
+    await nextFrame(el);
```
Comment (Sakchai-Refinitiv) [inline packages/translate/__test__/elf-translate-element.test.js:50]: Should remove this line

File: packages/translate/__test__/elf-translate.navigator.test.js [hunk 6]
```diff
@@ -8,16 +8,10 @@ describe('Elf Translate Navigator Test', () => {
...
-    }
-    else {
-      // Force navigator language
-      Object.defineProperty(navigator, 'language', { value: 'ru' });
-      el = await fixture('<test-translate></test-translate>');
-      expect(el.defaultEl.innerText).to.equal('Региональные настройки: ru', 'Navigator locale is not taken into account');
-    }
+    // Force navigator language
+    Object.defineProperty(navigator, 'language', { value: 'ru' });
+    el = await fixture('<test-translate></test-translate>');
```
Comment (Sakchai-Refinitiv) [inline packages/translate/__test__/elf-translate.navigator.test.js]: let or const for `el` should not declare on this line

File: packages/translate/__test__/elf-translate.navigator.test.js [hunk 7]
```diff
@@ -26,7 +20,7 @@ describe('Elf Translate Navigator Test', () => {
 
     el.lang = 'en-GB';
     await elementUpdated(el);
-    await nextFrame(el); // need for IE11
+    await nextFrame(el);
```
Comment (Sakchai-Refinitiv) [inline packages/translate/__test__/elf-translate.navigator.test.js:21]: Should remove this line

File: packages/elements/src/overlay/__test__/managers/focus-manager.test.js (hunk 2 above)
Comment (bualoy-napat) [reply packages/elements/src/overlay/__test__/managers/focus-manager.test.js:76]: Yes, it works well on modern browsers

File: packages/translate/__test__/elf-translate.navigator.test.js (hunk 6 above)
Comment (bualoy-napat) [reply packages/translate/__test__/elf-translate.navigator.test.js]: fixed

**Review by Sakchai-Refinitiv — APPROVED** [review]
Good

**Pull request merged**